tests/integration/server.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
+
+from google.cloud import logging as gcloud_logging
```

Comment:
This is a legitimate name collision, which is the only good reason to alias.  So I would say the alias is ok here.